scan/lib/scan/runner.rb

```diff
@@ -80,8 +80,16 @@ def handle_results(tests_exit_status)
     end
 
     def test_results
-      return "" unless Scan.cache[:temp_junit_report]
```

Comment:
Still trying to decide if `return ""` is a necessary path in this method now.

Based on the old code ([source](https://github.com/fastlane/fastlane/blob/a1d5794c0189b3a12a45ae6bb60f3eb245473e21/scan/lib/scan/runner.rb)), the original implementation was not concerned with `output_file` being missing or its output being bad, which makes sense to me.